map/src/data/markers.ts

```diff
@@ -263,17 +271,23 @@ const LOCATIONS = {
       lng: -72.717293,
       serviceRadius: 132230,
     },
+    VA_AUGUSTA: {
+      description: 'Augusta, Virginia',
+      lat: 38.14915,
+      lng: -79.122097,
+      serviceRadius: 32670,
+    },
     VA_NORFOLK: {
       description: 'Norfolk, Virginia',
       lat: 36.846224,
       lng: -76.284361,
       serviceRadius: 32670,
     },
-    VA_AUGUSTA: {
```

Comment:
We  shouldn't be removing existing locations as other map points do depend on them.

Reply:
sorry it got moved here https://github.com/reach4help/reach4help/pull/213/files/4f6ea499bdd61b202eb6d2e649b31572038ea1bd#r400551469, i was trying to alphabetize this so folks dont duplicate